osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -7,6 +6,42 @@
 from osmaxx.excerptexport import models
 
 
+def module_converter_configuration(name, export_formats, export_options):
+    """
+    :param export_formats example:
+        {
+            'txt': {
+                'name': 'Text',
+                'file_extension': 'txt',
+                'mime_type': 'text/plain'
+            },
+            'markdown': {
+                'name': 'Markdown',
+                'file_extension': 'md',
+                'mime_type': 'text/markdown'
+            }
+        }
+    :param export_options example:
+        {
+            'image_resolution': {
+                'label': 'Resolution',
+                'type': 'number',
+                'default': '500'
+            },
+            'quality': {
+                'label': 'Quality',
+                'type': 'number',
+                'default': '10'
+            }
+        }
+    """
+    return {
```

Comment:
>The function's signature enforces presence of expected components

No, they don't, they still can be passed as `None` and everything looks fine. I'd argue it's even worse because an exception doesn't come up at the right place: since the dictionary has that item defined an is None it will raise some other exception at (probably quite unexpected) place, instead of raising a KeyError at the very instance of access.

I'd rather have a constant to make it clear, that this is a, well, (config) constant.